ansible/AnsibleSkeleton.groovy

```diff
@@ -1,275 +0,0 @@
-class AnsibleSkeleton {
```

Comment:
Can we leave this file in the repo for future use (generates the basic structure for new ansible scripts)?

We should remove ```tomcat_webapps={{tomcat_dir}}/webapps``` from line 242, and add ```groupId: ala.org.au``` at line 73, too.